src/isomorphic/children/__tests__/ReactChildren-test.js

```diff
@@ -165,35 +151,30 @@ describe('ReactChildren', function() {
     var mappedChildren =
       ReactChildren.map(instance.props.children, callback);
     expect(callback.calls.length).toBe(5);
-    expect(ReactChildren.count(mappedChildren)).toBe(5);
+    expect(ReactChildren.count(mappedChildren)).toBe(4);
```

Comment:
Minor behavior change: though `ReactChildren.map()` still calls the callback with null values, the result of `map()` omits them, so `count(map([1, <two />, null], id))` returns 2 now instead of 3.